File: cpp/Leetcode/Python/problem80.py
class Solution:
    def removeDuplicates(self, nums) -> int:
        index=1
        occurence=1
        for i in range(1,len(nums)):
            if(nums[i]==nums[i-1]):
                occurence+=1
            else:
                occurence=1
            if(occurence<=2):
                nums[index]=nums[i]
                index=index+1
        return index

File: cpp/Leetcode/c++/problem80.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int index=1,occurence=1;
        for(int i=1;i<nums.size();i++){
            if(nums[i]==nums[i-1]){
                occurence+=1;
            }else{
                occurence=1;
            }
            if(occurence<=2){
                nums[index]=nums[i];
                index+=1;
            }
        }
        return index;
    }
};